src/periflow_sdk/utils.py

```diff
@@ -1,83 +1,21 @@
 """Helper functions
 """
-import os
-from dataclasses import dataclass
+import functools
+from enum import Enum
 
 
-_AXIS_TO_DEGREE = {
-    x: f"{x.upper()}_DEGREE" for x in ["pp", "dp", "mp"]
-}
+class SaveType(str, Enum):
+    NORMAL = "NORMAL"
+    EMERGENCY = "EMERGENCY"
 
 
-_AXIS_TO_RANK_ATTR = {
-    x: f"{x}_rank" for x in ["pp", "dp", "mp"]
-}
-
-
-@dataclass
-class DistributeConfig:
-    local_rank: int
-    rank: int
-
-    def __post_init__(self):
-        self.pp_rank: int = 0
-        self.mp_rank: int = 0
-        self.dp_rank: int = 0
+@functools.wraps
+def check_initialized(func):
+    def wrapper(training_manager, *args, **kwargs):
+        assert training_manager.has_initialized, f'{func.__name__} must be called after pf.init()!'
+        return func(*args, **kwargs)
+    return wrapper
```

Comment:
```suggestion
def check_initialized(func):
    @functools.wraps
    def wrapper(training_manager, *args, **kwargs):
        assert training_manager.has_initialized, f'{func.__name__} must be called after pf.init()!'
        return func(*args, **kwargs)
    return wrapper
```

To preserve the name and docstrings of decorated function, `wraps` needs to decorate the inner function `wrapper`.